ivy/data_classes/array/experimental/norms.py

```diff
@@ -39,93 +39,123 @@ def l2_normalize(
         """
         return ivy.l2_normalize(self, axis=axis, out=out)
 
+    def batch_norm(
+        self,
```

Comment:
Type-hints are missing for some `self` arguments.

Reply:
Updated :+1: